cli_tools/gce_vm_image_import/main.go

```diff
@@ -128,10 +132,13 @@ func validateAndParseFlags() error {
 	}
 
 	if *sourceFile != "" {
-		_, _, err := storageutils.SplitGCSPath(*sourceFile)
+		bucketName, objectName, err := storageutils.SplitGCSPath(*sourceFile)
```

Comment:
if declare err var first, you can avoid having these two temporary variables. You probably also don't need to have sourceBucketName/sourceObjectName be pointers, they can be plain strings

Reply:
Having args to be populated as pointers is how other args in this program did, so I just followed the pattern. I don't see a obvious pro/con on pointers v.s. objects here.

I can replace these 2 temp var with sourceBucketName/sourceObjectName, but I chose not to, because it's meaningless to change their value when SplitGCSPath fails. It's an unexpected side effect, even though it's harmless.

Reply:
Other flags are pointers as the are returned from flag methods. This is not a flag but a regular variable so there's no reason to be a pointer. 

It's possible that some other regular variables were declared as pointers but there's no reason for it.

Reply:
I read the code carefully and got the hidden reason why those existing vars need to be a pointer: they need to call "testutils.SetStringP" to set/recover the value. So it's because of testability.
Due that my tests don't need to set them, I changed them to objects. Thanks!